src/components/Layout/Layout.tsx

```
@@ -1,50 +1,28 @@
 import type { ReactProps } from "src/types/types";
-import { motion } from "framer-motion";
 import { usePathname } from "next/navigation";
 import React from "react";
-import styled, { ThemeProvider } from "styled-components";
-
 import Footer from "@/components/Footer";
 import Header from "@/components/Header";
-import { ContainerStyle, GlobalStyles, MainStyles } from "../../styles/global-styles";
-import { theme } from "../../styles/theme";
 
 const variants = {
```

ESLint

[error] 'variants' is assigned a value but never used. Allowed unused vars must match /^_/u.

(unused-imports/no-unused-vars)